src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final int MIN_LENGTH = 9;
+
+    public static boolean validate(String password, String repeatPassword)
```

Comment:
why have you made this method static?

Reply:
Потому что класс не хранит состояния внутри себя, он только выполняет проверку.
Нам не имеет смысла создавать объект данного класса.

Reply:
it's a bad practice to use static methods, that makes your classes less flexible. So in general you need to avoid it even if a class doesn't have state

Reply:
Не согласен, но я как с подводной лодки - куда денусь.
Мое мнение строиться по двум причинам:
1. в задании четко указано 1 класс/1 метод/без полей.
2. Если бы плохой практикой было использование статических методов, почему тогда их "полного" в стандартных доках.

Reply:
Cause in java for now you use only core base classes and you build simple methods/classes. Whenever you start building something more complex you will use architecture patterns and complicated construction that consist of several classes with methods etc. So even if the class doesn't have any state (fields) using static methods won't allow you to use polymorphism and will make unit testing  of classes and methods more comlicated. And here using static keyword doesn't give any benefit at all but in future may cause you problems. But it's up to you, if you're sure in your decision - use stati method